pkg/controllers/securitypolicy/securitypolicy_controller.go

```diff
@@ -91,7 +93,14 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	// Use once.Do to ensure gc is called only once
 	common.GcOnce(r, &once)
 
-	obj := &v1alpha1.SecurityPolicy{}
+	var obj client.Object
+	if r.Service.NSXConfig.EnableVPCNetwork {
+		obj = &crdv1alpha1.SecurityPolicy{}
+		EnableVPCNetwork = true
```

Comment:
Is it guaranteed that other code relies on `EnableVPCNetwork` are always executed after `SecurityPolicyReconciler.Reconcile` is called?

Reply:
r.Service.NSXConfig.EnableVPCNetwork is initialized after loading config, EnableVPCNetwork is used only in https://github.com/vmware-tanzu/nsx-operator/blob/main/pkg/controllers/securitypolicy/securitypolicy_controller.go#L380-L380

Reply:
It doesn't really answer the question. I'm asking `EnableVPCNetwork`, not `r.Service.NSXConfig.EnableVPCNetwork`. 

In another word, is it guaranteed `reconcileSecurityPolicy` will only be executed after `SecurityPolicyReconciler.Reconcile`?

Reply:
Fixed.